common/test/acceptance/pages/lms/courseware.py

```diff
@@ -128,6 +128,36 @@ def start_proctored_exam(self):
         # elf.wait_for_element_presence(".proctored-exam-code", "unique exam code")
 
     @property
+    def entrance_exam_status_msg(self):
+        """
+        Return the entrance exam status message selector on the top of courseware page.
+        """
+        return self.q(css='#content .container section.course-content .sequential-status-message')
+
+    @property
+    def is_not_passed_message_present(self):
```

Comment:
We admittedly don't have a convention for properties (ie, field accessors) and when to use them vs. get methods.  However, this seems odd to have for a property -- asking an object to answer a question about its state seems to be an operation, versus simply returning the value of one of its fields.